src/aap_eda/core/validators.py

```diff
@@ -68,3 +69,14 @@ def check_awx_tokens(user_id: int) -> int:
         )
 
     return user_id
+
+
+def is_extra_var_dict(extra_var: str):
+    try:
+        data = yaml.safe_load(extra_var)
```

Comment:
@Dostonbek1 How is an empty string handled? Does the UI send in an empty string or None?

Reply:
Serializer has it set to `required=True` so passing an empty string or `null` will return the following error messages
```
{
  "extra_var": [
    "This field may not be blank."
  ]
}
```
and 
```
{
  "extra_var": [
    "This field may not be null."
  ]
}
```

Reply:
@lgalis Is the UI always sending some data in the extra_var?

Reply:
@Dostonbek1 , @mkanoor - if the Variables field is empty in the Activation form, the /extra_vars POST will not be called.